programs/beautysh.nix

```
@@ -0,0 +1,26 @@
+{ lib, pkgs, config, ... }:
+let
+  cfg = config.programs.beautysh;
+in
+{
+  options.programs.beautysh = {
+    enable = lib.mkEnableOption "beautysh";
+    package = lib.mkPackageOption pkgs "beautysh" { };
+    indent_size = lib.mkOption {
+      type = lib.types.nullOr lib.types.int;
```

Let's keep this simple and always pass the indent.
```suggestion
      type = lib.types.int;
```